website/src/components/code-block.js

```diff
@@ -4,23 +4,17 @@ import Highlight, { defaultProps } from 'prism-react-renderer';
 import { customProperties } from '@commercetools-frontend/ui-kit';
 import * as colors from '../colors';
 
-// TODO: improve colors
 const theme = {
```

Comment:
The changes in the theme colors are based on the `github` theme from `prism-react-renderer`